Sources/RoktUXHelper/UI/RoktLayoutUIView.swift

```diff
@@ -170,37 +124,23 @@ extension RoktLayoutUIView: LayoutLoader {
     ///   - injectedView: A closure returning the SwiftUI view to embed.
     public func load<Content>(onSizeChanged: @escaping ((CGFloat) -> Void),
                               injectedView: @escaping () -> Content) where Content: View {
-        cleanupEmbeddedView()
-
+        roktEmbeddedSwiftUIView?.removeFromSuperview()
         let vc = ResizableHostingController(rootView: AnyView(injectedView()))
-        let swiftuiView = vc.view!
-        self.roktEmbeddedSwiftUIView = swiftuiView
+        guard let swiftUIView = vc.view else { return }
 
+        self.roktEmbeddedSwiftUIView = swiftUIView
         parentViewControllers?.addChild(vc)
-        swiftuiView.translatesAutoresizingMaskIntoConstraints = false
-
-        decideTranslatesAutoresizingMask()
-
-        addSubview(swiftuiView)
-
-        self.frame = CGRect(x: self.frame.minX, y: self.frame.minY, width: self.frame.width, height: 0)
-
-        addEmbeddedLayoutConstraints(embeddedView: swiftuiView)
-
+        swiftUIView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(swiftUIView)
+        addEmbeddedLayoutConstraints(embeddedView: swiftUIView)
         vc.didMove(toParent: parentViewControllers)
     }
 
     /// Updates the size of the embedded view.
     /// - Parameter size: The new height for the embedded view.
     public func updateEmbeddedSize(_ size: CGFloat) {
         if roktEmbeddedSwiftUIView != nil {
-            for cons in self.constraints where cons.firstAttribute == NSLayoutConstraint.Attribute.height {
```

Comment:
What if parent view, have a height LayoutConstraint = 0?

Reply:
parent view's constraints will exist in parentView.constraints, not self.constraints.